specification/translation/Azure.AI.DocumentTranslation/main.tsp

```diff
@@ -0,0 +1,150 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+@useAuth(ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key">)
+@service({
+  title: "Azure.AI.DocumentTranslation",
+})
+@versioned(DocumentTranslation.Versions)
+namespace DocumentTranslation;
+
+/** The Azure.AI.DocumentTranslation versions. */
+enum Versions {
+  /** The 2023-11-01-preview version of the Azure.AI.DocumentTranslation. */
+  @useDependency(Azure.Core.Versions.v1_0_Preview_1)
+  `2023-11-01-preview`,
+}
+
+alias CommonParameters = {
+  /** Mandatory API version parameter */
+  @query("api-version")
+  apiVersion: string;
+
+  /** The value is the region of the translator resource. This value is optional if when custom endpoint is used.
+   * The value can be found in Azure portal. Translator resource => Resource Management => Keys and Endpoint => Location/Region  */
+  @header("Ocp-Apim-Subscription-Region")
+  subscriptionRegion?: string;
+
+  /** A client-generated GUID to uniquely identify the request. */
+  @header("X-ClientTraceId")
+  clientTraceId?: string;
+
+  /** Etag comparison, if it is provided and its value matches the response then request will return Status => 304 (Not modified). */
+  @header("If-None-Match")
+  ifNoneMatch?: string;
+};
+
+alias CommonResultHeaders = {
+  /** Value generated by the service to identify the request. It is used for troubleshooting purposes. */
+  @header("X-RequestId")
+  requestId: string;
+
+  /** This will have value of response hash, customer can provide this value for any consecutive request as “If-None-Match” header. To make sure get response if the value is changed otherwise 304. */
+  @header("ETag")
+  eTag: string;
```

Comment:
nit, `...EtagResponseEnvelope` https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/lib/models.tsp#L162-L168

Reply:
ETag is not needed for this API so removing the ETag.